### PR TITLE
Feature: format notification email date and time by user locale 

### PR DESCRIPTION
**Pull request by samanehsan**
<h3>Purpose</h3>
At the moment, the times in notification emails are only shown in 24-hour time. Also, the date is always shown as the month before the day: `17:30 on Tuesday, March 3 EST`. Show the time/date as it would be displayed in the user's locale. For example, a user in Charlottesville would see `11:05 AM on Tuesday, March 3, 2015` but a user in Berlin would see `17:05 on Dienstag, 3. März 2015`. 

<h3>Changes</h3>
Get user locale data from `window.navigator.language` or `window.navigator.userLanguage` (for IE), store on the user model and use to format date and time with the Babel library.

### CODE REVIEW

**Comment by lbanner** [comment]
@samanehsan -- Do you know if there was an Issue submitted for this?

**Comment by samanehsan** [comment]
I don't there was! Also, I think I'm going to have to close this PR too and branch off of develop instead of master.

**Comment by lbanner** [comment]
OK - thanks

**Comment by sloria** [comment]
@samanehsan No need to create a new branch off of master. This PR will do. :+1:

**Comment by lbanner** [comment]
:confetti_ball:

**Comment by samanehsan** [comment]
Ok! Thanks @sloria!

File: website/profile/views.py [hunk 1]
```diff
@@ -87,6 +87,9 @@ def update_user(auth):
     # TODO: Expand this to support other user attributes
     if 'timezone' in data:
         user.timezone = data['timezone']
+    if 'locale' in data:
```
Comment (sloria) [inline website/profile/views.py]: What if `data['locale']` is `None` or an empty string?

**Comment by sloria** [comment]
New views tests should be added for the new input to `update_user` .

**Comment by sloria** [comment]
Pass finished. :point_up:

File: website/profile/views.py (hunk 1 above)
Comment (samanehsan) [inline website/profile/views.py]: Good catch. I can also handle the case where `data['timezone']` is `None` or an empty string.

**Comment by samanehsan** [comment]
Done responding to second pass @sloria :tada: 

This branch is off of master and GH says it can't automatically merge...should I pull in develop?

**Comment by sloria** [comment]
Looks good. :octopus: I'll take care of the merge.

**Pull request merged**